.github/workflows/ci.yaml

```diff
@@ -2,8 +2,6 @@ name: ci
 on:
   pull_request:
     paths-ignore:
-    - '.github/workflows/docker.yaml'
```

Comment:
why are we removing the first two workflow files but not `'.github/workflows/release.yaml'`?

Reply:
The release isn't required when running PR checks.